src/test/kotlin/validation/AuthenticationValidatorTest.kt

```diff
@@ -0,0 +1,144 @@
+package se.uulm.snowballr.backend.validation
+
+import `in`.rcard.assertj.arrowcore.EitherAssert
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import se.uulm.snowballr.backend.assertInvalidResult
+import se.uulm.snowballr.backend.model.InvalidEmail
+import se.uulm.snowballr.backend.validation.AuthenticationValidator.validateRegisterRequest
+import snowballr.Authentication
+
+class AuthenticationValidatorTest {
+    @Nested
+    inner class RegisterRequest {
+        private val validRegisterRequestBuilder: Authentication.RegisterRequest.Builder =
+            Authentication.RegisterRequest
+                .newBuilder()
+                .setEmail("alice.smith@example.com")
+                .setFirstName("Alice")
+                .setLastName("Smith")
+                .setPassword("AAbb__00")
+
+        @Test
+        fun `When a valid request is validated, then no issue is returned`() {
+            val request = validRegisterRequestBuilder.build()
+            val result = validateRegisterRequest(request)
+
+            EitherAssert.assertThat(result).isRight()
+        }
+
+        @Test
+        fun `When an invalid email is validated, then the 'InvalidEmail' issue is returned`() {
+            val request =
+                validRegisterRequestBuilder
+                    .setEmail("invalid-email")
+                    .build()
+            val result = validateRegisterRequest(request)
+
+            assertInvalidResult(result, InvalidEmail::class.java)
+        }
+
+        @Test
+        fun `When a blank first name is validated, then the 'BlankField' issue is returned`() {
+            val request =
+                validRegisterRequestBuilder
+                    .setFirstName("")
+                    .build()
+            val result = validateRegisterRequest(request)
+
+            assertInvalidResult(result, se.uulm.snowballr.backend.model.BlankField::class.java)
+        }
+
+        @Suppress("MaxLineLength", "ktlint:standard:max-line-length")
+        @Test
```

Comment:
```suggestion
        @Test
```
Same for the other cases.

Just noticed that this is still reported. Deactivate the style rule `MaxLineLength`. The formatting rule has the right configuration.

Reply:
I'm not quite sure what you mean.

Reply:
You can remove these suppressions since we changed the linter/formatter. But, there are two rules that check for the maximum line length: [`MaxLineLength`](https://detekt.dev/docs/rules/style#maxlinelength) and [`MaximumLineLength`](https://detekt.dev/docs/rules/style#maxlinelength). The latter one is from ktlint and ignores methods with backticks, which is what we want here. As stated in the latter, we can deactivate the first rule and the second one will still cover all cases. As an alternative, we could exclude the whole test directory for the first rule because we only use methods with backticks in there.

Reply:
Like this?

Reply:
Yes